pkg/eventpersister/persister/persister-dwh.go

```diff
@@ -0,0 +1,597 @@
+// Copyright 2022 The Bucketeer Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package persister
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"github.com/golang/protobuf/proto"
+	"github.com/golang/protobuf/ptypes"
+	"github.com/golang/protobuf/ptypes/wrappers"
+	"go.uber.org/zap"
+	"golang.org/x/sync/singleflight"
+
+	"github.com/bucketeer-io/bucketeer/pkg/errgroup"
+	"github.com/bucketeer-io/bucketeer/pkg/eventpersister/datastore"
+	ec "github.com/bucketeer-io/bucketeer/pkg/experiment/client"
+	featuredomain "github.com/bucketeer-io/bucketeer/pkg/feature/domain"
+	featurestorage "github.com/bucketeer-io/bucketeer/pkg/feature/storage"
+	"github.com/bucketeer-io/bucketeer/pkg/health"
+	"github.com/bucketeer-io/bucketeer/pkg/pubsub/puller"
+	"github.com/bucketeer-io/bucketeer/pkg/pubsub/puller/codes"
+	"github.com/bucketeer-io/bucketeer/pkg/storage/v2/bigquery/query"
+	bigtable "github.com/bucketeer-io/bucketeer/pkg/storage/v2/bigtable"
+	eventproto "github.com/bucketeer-io/bucketeer/proto/event/client"
+	ecproto "github.com/bucketeer-io/bucketeer/proto/eventcounter"
+	exproto "github.com/bucketeer-io/bucketeer/proto/experiment"
+	featureproto "github.com/bucketeer-io/bucketeer/proto/feature"
+)
+
+var (
+	twentyFourHours = 24 * time.Hour
+)
+
+type PersisterDwh struct {
```

Comment:
```suggestion
type PersisterDWH struct {
```

https://github.com/golang/go/wiki/CodeReviewComments#initialisms